action.yml

```diff
@@ -146,19 +146,8 @@ runs:
     - name: Set up env vars (Linux)
       if: ${{ (runner.os == 'Linux') }}
       run: |
-        set -x
-        if command -v dnf > /dev/null 2>&1; then
-          echo "Detected mariner / hyperv"
-          echo "HYPERV_SHOULD_BE_PRESENT=true" >> $GITHUB_ENV
-          echo "HYPERV_SHOULD_HAVE_STABLE_API=false" >> $GITHUB_ENV
-          ls -al /dev/mshv
-          whoami
-        else
-          echo "Detected Ubuntu / kvm"
-          echo "KVM_SHOULD_BE_PRESENT=true" >> $GITHUB_ENV
-          sudo ls -al /dev/kvm
```

Comment:
Think we still need these 3 lines unless it is done in the base image?

Reply:
I'm not sure I understand why?

Reply:
Not sure if I missed the `sudo adduser $(whoami) kvm` in the first review, anyway I think this should solve it